vagga.yaml

```diff
@@ -0,0 +1,26 @@
+containers:
+  pip-freeze:
+    setup:
+      - !Alpine v3.4
+      - !Py3Install
+        - pip
+        - Django
```

Comment:
Normally you would need to lock some range of versions, because major version change may introduce backward incompatible changes.